### PR TITLE
Testing annoy console

### PR DESCRIPTION
**Pull request by Sajal008**
test: ✅ annoy unit test

### CODE REVIEW

File: tests/test_annoy_console.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Unit this for annoy
+# Import slogpy from slog
+from slogpy.slog import Slog as slog  # noqa: N813, E402
```
Comment (neel-m) [inline tests/test_annoy_console.py]: the error message (that you are ignoring) E402 is there for a reason.
https://docs.astral.sh/ruff/rules/module-import-not-at-top-of-file/

It is giving you this error message because you are not following the standard  (PEP8) style of imports at the top of the file.  Most likely it is giving you this, because your have a mix of comments and docstrings with the imports in between them.

File: tests/test_annoy_console.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Unit this for annoy
+# Import slogpy from slog
```
Comment (neel-m) [inline tests/test_annoy_console.py]: These comments are of low value and can be removed.

File: tests/test_annoy_console.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+from slogpy.slog import Slog as slog  # noqa: N813, E402
+
+"""This module contains tests for the Slog class, specifically the annoying method,
+which is responsible for slog annoying messages.
+"""
```
Comment (neel-m) [inline tests/test_annoy_console.py]: This doc string should be moved to the top of the file to eliminate the need to ignore E402

File: tests/test_annoy_console.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+# (note to self) make sure the count is only once......
```
Comment (neel-m) [inline tests/test_annoy_console.py]: do not leave notes to (future/current/past) self in production code

File: tests/test_annoy_console.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+def test_annoy_unique_message(capsys, msg="This is a annoy test"):  # noqa: D103
+    """Timestamp checking method
```
Comment (neel-m) [inline tests/test_annoy_console.py]: I don't see any timestamp being checked and that is not the name of the function, nor does it actually appear to be checking a timestamp.

File: tests/test_slogpy_console.py [hunk 6]
```diff
@@ -0,0 +1,16 @@
+"""Import slogpy from slog"""
```
Comment (neel-m) [inline tests/test_slogpy_console.py:1]: This is not a valuable docstring for this file/module.  This is the place where you should explain the reason why you created this file, i.e. what were you thinking when you created it and why did you do it, what do you think someone should be thinking about when they look at this code.

Since there are now other unit tests that test info(), I would just delete this whole file.

File: tests/test_annoy_console.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+    Given: a single line annoy message
+    When: passed into slog.annoy
+    Then: method will check for annoy message
+    """
```
Comment (neel-m) [inline tests/test_annoy_console.py:11]: The wording here is hard to follow.  It also doesn't really capture the value proposition of annoy().  The reason a developer should use annoy() is to remind themselves that there is more work to be done and they don't want to forget about it, but they also don't want to be overly annoyed by it.  annoy() does this by only displaying the message once.  Looking at the code, it also only puts it in the log once (for each unique message).

File: tests/test_annoy_console.py [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    # Arrange
+
+    # Act
+    slog.annoy(msg)
+
+    # Assert (note to self: )
+    # Check if the message is in the _annoyed dictionary
+    assert msg in slog._annoyed
```
Comment (neel-m) [inline tests/test_annoy_console.py]: Why is this check here?
It is checking implementation details that will couple the implementation with the test.  There does not appear to be any reason to do this.

**Review by neel-m — CHANGES_REQUESTED** [review]

**Comment by Sajal008** [comment]
This PR is no longer needed
